src/bootstrap/bootstrap.py

```diff
@@ -1159,7 +1132,7 @@ def bootstrap(help_triggered):
     # Fetch/build the bootstrap
     build.download_toolchain()
     # Download the master compiler if `download-rustc` is set
-    build.maybe_download_ci_toolchain()
+    # build.maybe_download_ci_toolchain()
```

Comment:
I would prefer we delete rather than comment out call sites for unused functions -- git history will have them anyway.

Reply:
Good catch, thanks :) I completely missed removing the dead code from bootstrap, forgot to delete `maybe_download_ci_toolchain` as well.